app/utils/callback/lava.py

```diff
@@ -394,6 +364,34 @@ def _get_lava_bootloader_meta(self, bl_meta):
                     meta[meta_key] = v
         return meta
 
+    def _get_directory_path(self):
+        """Create the dir_path from LAVA metadata
+
+        Update the metadata with the storage path of the artifacts.
+        If possible, use the file_server_resource from the metadata.
+
+        :param meta: The boot meta-data.
+        :type meta: dictionary
```

Comment:
This method doesn't have any arguments.

In fact I think it's probably OK to not have docstrings for private methods.